### DISCUSSION TITLE
Claiming empty and unused organization

### DISCUSSION THREAD

**Discussion by LodeKennes** [community/community#33573] [Other Feature Feedback, Questions, & Ideas]
Hi

I've worked for an IT Consultancy for the past 4 years. Recently we've been thinking about open-sourcing some of our software and wanted to create an organization on Github. Yet we noticed our organization name has already been claimed. 

There are no public members, yet the public repo's (they don't have commits) lead us to believe that the organization was created by an (ex-)employee.

Is there any way to find the identity of the user? I can prove working for the organization if necessary.

Kind regards
Lode Kennes

**Comment by mortenseifert** [comment]
@LodeKennes did you ever succeed in getting the organization handed over?

**Reply by mortenseifert** [reply]
I found this https://github.com/orgs/community/discussions/23468, which may be of some help.

**Reply by LodeKennes** [reply]
Hi, we've figured it out. A colleague of mine was able to contact someone to claim the organization. Thank you anyway!

**Reply by mortenseifert** [reply]
We were unable to claim the organization - only if it was related to trademark, would support help, so we have appended our country code to the organisation name.
